connaisseur/flask_application.py

```diff
@@ -24,6 +24,7 @@
 """
 CONFIG = Config()
 DETECTION_MODE = os.environ.get("DETECTION_MODE", "0") == "1"
+VALIDATION_MODE_MUTATE = os.environ.get("VALIDATION_MODE_MUTATE", "true") == "true"
```

Comment:
This falls back silently to other mode if config is different, i.e. if "True". As long as we don't validate that value is `true` or `false`, I'd rather have the fallback in favour of the default, i.e.
```suggestion
VALIDATION_MODE_MUTATE = os.environ.get("VALIDATION_MODE_MUTATE", "true") != "false"
```

Reply:
ah...good catch. Will change